src/show/pages/public/Explore/index.js

```diff
@@ -0,0 +1,89 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Button, ProgressBar } from '../../../components';
+
+const Explore = () => {
+  const [projectCard, setProjectCard] = useState([]);
```

Comment:
Why `projectCard`? Not a good variable name.

Reply:
did it